rpc/client.go

```diff
@@ -191,6 +195,45 @@ func (c *Client) Healthy() <-chan struct{} {
 	return c.healthy.Load().(chan struct{})
 }
 
+func (c *Client) isHealthy() bool {
+	select {
+	case <-c.Healthy():
+		return true
+	default:
+	}
+	return false
+}
+
+// WaitHealthy blocks until the channel is healthy, a health indication has
```

Comment:
This comment should emphasize the return value: "WaitHealthy returns the health of the Client. On the first connection of a newly-created Client, WaitHealthy will block for up to Context.HealthWait if its health has not yet been determined."